wire/msggetcfilters.go

```diff
@@ -0,0 +1,129 @@
+// Copyright (c) 2017 The btcsuite developers
+// Copyright (c) 2017 The Lightning Network Developers
+// Copyright (c) 2019 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package wire
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/decred/dcrd/chaincfg/chainhash"
+)
+
+// MaxGCFiltersPerMsg is the maximum number of committed filter requests
+// allowed per message.
+const MaxGCFiltersPerMsg = 500
```

Comment:
This probably should be lower than 128, every `MsgCFilter` has a max size of 256KB (`MaxCFilterDataSize`), so you don't want to build messages that have more than messages maximum payload size (`MaxMessagePayload` 32MB).

Reply:
good catch, resolving.